client/my-sites/stats/activity-log-day/index.jsx

```diff
@@ -116,20 +119,25 @@ class ActivityLogDay extends Component {
 		return (
 			<div>
 				<div className="activity-log-day__day">
+					{ /* eslint-disable indent */ }
```

Comment:
I've never seen this trick of putting js comments inside otherwise empty braces, cool 👍.

I'm not sure the eslint disable is necessary though, because it seems the code below is indented too far? Also, if it is getting hairy with indents, we can always move the translations above the jsx and stash them in `const`s.

Reply:
I pulled this off of another PR, these lines are `prettier` formatted and `eslint` errors 😩 .

Pulling out the translations into `const` declarations is a good alternative 👍

Reply:
Forget prettier, I just cleaned it up in 0b7f89b